client/src/entrypoints/admin/page-editor.js

```diff
@@ -177,6 +180,11 @@ function InlinePanel(opts) {
 
       self.updateMoveButtonDisabledStates();
       self.updateAddButtonState();
+      initCollapsiblePanels(
+        document.querySelectorAll(
+          `#inline_child_${newChildPrefix} [data-panel-toggle]`,
+        ),
+      );
```

Comment:
Not a fan of having to import and call this, this should hopefully no longer be needed if we proceed with https://github.com/wagtail/rfcs/pull/78.